test/helpers/xcm.ts

```diff
@@ -874,3 +868,15 @@ export const registerXcmTransactorDerivativeIndex = async (context: DevModeConte
       .tx.sudo.sudo(context.polkadotJs().tx.xcmTransactor.register(ALITH_ADDRESS, 0))
   );
 };
+
+export const expectXcmEventMessage = async (context: DevModeContext, message: string) => {
+  const records = await context.polkadotJs().query.system.events();
+
+  const filteredEvents = records
+    .map(({ event }) => (context.polkadotJs().events.xcmpQueue.Fail.is(event) ? event : undefined))
+    .filter((event) => event);
+  expect(filteredEvents).to.have.lengthOf(1);
```

Comment:
Not really sure if we should have expects here. What do you think @timbrinded ?

Reply:
I'm not huge fan of expects in test helpers. I'd prefer if it was instead returned boolean or number, and your test had an expect for `.toBe(1 OR true)` <depending on return type>

That way if theres a failure you can easily see where the test failed and not have to jump around into helpers too much